tdei-ui/src/services/axiosService.js

```diff
@@ -85,7 +85,7 @@ axios.interceptors.response.use(
       } catch (refreshError) {
         // If refreshing the token also fails with a 401, trigger the re-login modal
         console.log(originalRequest)
-        if ((refreshError.status === 401 || refreshError.response?.status === 401) && originalRequest.url.indexOf("/authenticate") === -1) {
+        if ((refreshError.status === 401 || refreshError.response?.status === 401) && originalRequest.url.indexOf("/authenticate") === -1 && window.location.pathname !== "/login") {
```

Comment:
Check condition with indexOf , as there is possibility query params will be attached to /login url

Reply:
Resolved